app/models/ability.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class Ability
+  include CanCan::Ability
```

Comment:
@sansuaza Please explain what this **include** method does?

Reply:
this method brings the Ability module that is inside the CanCan module